src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-form.tsx

```diff
@@ -141,7 +149,7 @@ export const CharacteristicsForm = ({ previousValues, isModification = false })
             name={SWITCHED_ON_SUSCEPTANCE}
             label={'SwitchedOnMaxSusceptance'}
             adornment={SusceptanceAdornment}
-            previousValue={previousValues?.bperSection * previousValues?.sectionCount}
+            previousValue={previousValues && previousValues?.bPerSection * previousValues?.sectionCount}
```

Comment:
Same there is no need of question mark when you already checked if the value exists.